src/screens/SendConfirmationSheet.tsx

```diff
@@ -289,6 +301,82 @@ export default function SendConfirmationSheet() {
         ]
   );
 
+  useEffect(() => {
+    if (isENS) {
+      const promises = [
+        estimateGasLimit({
+          address: accountAddress,
+          amount: 0,
+          asset: asset,
+          recipient: toAddress,
+        }),
+      ];
+      const sendENSOptions = Object.fromEntries(
+        checkboxes.map(option => [option.id, option.checked])
+      );
+      if (sendENSOptions['clear-records']) {
+        let records = Object.keys({
+          ...(ensProfile?.data?.coinAddresses || {}),
+          ...(ensProfile?.data?.records || {}),
+        }).reduce((records, recordKey) => {
+          return {
+            ...records,
+            [recordKey]: '',
+          };
+        }, {});
+        if (sendENSOptions['set-address']) {
+          records = { ...records, ETH: toAddress };
+        } else {
+          records = { ...records, ETH: AddressZero };
+        }
+        promises.push(
+          estimateENSSetRecordsGasLimit({
+            name: asset?.name,
```

Comment:
i think this one could fail for ENS with non ASCII characters, if that´s the case we should clean the name before estimating gas

Reply:
just want to make it clear for people reviewing the PR (talking with ben about this internally), the problem is not the ASCII character is that opensea returns an alert icon together with the name
![image](https://user-images.githubusercontent.com/12115171/171298265-4a621c8a-2571-488a-b324-e746a0ff84f4.png)